packages/frontend-api/src/Model/Resolver/Products/ProductResolver.php

```diff
@@ -52,10 +52,6 @@ public function resolver(string $uuid): Product
             throw new UserError($productNotFoundException->getMessage());
         }
 
-        if ($product->isVariant() === true) {
-            throw new UserError('This product is variant and there is no support for variants yet.');
-        }
-
         return $product;
```

Comment:
Variable `$product` might be removed and replaced by `return $this->productFacade->getByUuid($uuid);` on line 50.

Reply:
:+1: